config/routes.rb

```diff
@@ -1,4 +1,18 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users, controllers: { registrations: 'users/registrations' }
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root to: "home#start"
+  # get "microposts", to: "microposts#index"
+  # get "microposts/new", to: "microposts#new", as: :new_microposts
```

Comment:
**[OPTIONAL]**
Why not make `http://localhost:3000/users/sign_up` the root for your page.